src/app/admin/_components/specialist/create/Details.js

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import React from 'react';
+import { NumberInput, required, SelectInput } from 'react-admin';
+import { FormatOfWork, Gender } from '@prisma/client';
+import { getChoicesList } from '@/app/admin/_utils/getChoicesList';
+import { FormFieldWrapper } from '@/app/admin/_components/FormFieldWrapper';
+import {
+  SpecialistCreateFormBlocks,
+  SpecialistFormFields,
+} from '@/app/admin/_lib/specialistData';
+
+const Details = () => {
+  const genderChoicesList = getChoicesList(Object.values(Gender));
+  const formatOfWorkChoicesList = getChoicesList(Object.values(FormatOfWork));
+
+  const { gender, yearsOfExperience, formatOfWork } = SpecialistFormFields;
+
+  return (
+    <FormFieldWrapper
+      title={SpecialistCreateFormBlocks.details}
+      className="mt-3"
+    >
+      <div className="flex w-full gap-6">
+        <SelectInput
+          name={gender.name}
+          source={gender.name}
+          label={gender.label}
+          validate={gender.validate && required()}
```

Comment:
I still dont understand why do we manually pass validation, shouldn't it be handled by react-hook-form + zod resolver?